aiohttp/helpers.py

```diff
@@ -219,14 +215,8 @@ def proxies_from_env():
     return ret
 
 
-if PY_352:
-    def create_future(loop):
-        return loop.create_future()
-else:
-    def create_future(loop):  # pragma: no cover
-        """Compatibility wrapper for the loop.create_future() call introduced in
-        3.5.2."""
-        return asyncio.Future(loop=loop)
+def create_future(loop):
+    return loop.create_future()
```

Comment:
You can remove this function at all and just use ‘loop.create_future’ everywhere

Reply:
:+1: